README.md

```diff
@@ -297,13 +297,41 @@ with the running machine (via SSH). It supports the following methods:
     Stops or disconnects the system. This stops running boxes or images and disconnects from
     running systems.
 
-  * `run_command(command, *args, options = {})`
-    `run_command(command_and_args, options = {})`
+  * `run(command, *args, options = {})`
+    `run(command_and_args, options = {})`
 
-    Executes a command on the running machine. The execution is implemented
-    using [Cheetah](https://github.com/opensuse/cheetah) and the `run_command`
-    method behaves mostly the same as
-    [`Cheetah.run`](http://rubydoc.info/github/openSUSE/cheetah/master/Cheetah.run).
+    Executes a command on the running machine. A VM::CommandResult instance is
+    returned which can be used with special RSpec matchers to define the
+    expected behavior. Examples:
+
+      # Expect a certain exit code
+      expect(@vm.run("ls -l")).to have_exit_code(0)
```

Comment:
In machinery we have one case where we call one command several times with different options. In some cases there is stderr output and in others there isn't, so neither `succeed` nor `succeed.with_stderr` work.

I introduced that method as a workaround for that problem, but I'm open for other ideas of course if you have any.